package.json

```diff
@@ -113,6 +113,7 @@
       "testRule": "readonly",
       "testRuleConfigs": "readonly"
     },
+    "reportUnusedDisableDirectives": true,
```

Comment:
> [For legacy compatibility, true is equivalent to "warn" and false is equivalent to "off".](https://eslint.org/docs/latest/use/configure/configuration-files#reporting-unused-disable-directives)

So if we ever use a flat config for ESLint it will have to be changed to `"error"`.